packages/core-components/src/components/alert/alert.scss

```diff
@@ -46,7 +46,6 @@
 
   &__content {
     display: flex;
-    align-items: center;
```

Comment:
should be baseline to avoid alignment shifts when there are buttons or other larger elements in the alert.